Cargo.lock

```diff
@@ -86,7 +86,7 @@ dependencies = [
  "cc",
  "cfg-if",
  "libc",
- "miniz_oxide",
+ "miniz_oxide 0.4.4",
```

Comment:
Looks like this dupes the dep on miniz_oxide 😕

Reply:
It doesn't actually duplicate it, because it's gated on `stdx/backtrace`, which we have disabled.

But we have a dupe of `parking_lot` (pulled in by `salsa`) and one of `hashbrown` (through `indexmap`).

Reply:
Ah okay